server/src/middleware/routes.test.ts

```diff
@@ -1275,4 +1274,139 @@ describe("Endpoints", () => {
       expect(res.status).toEqual(401)
     })
   })
+
+  describe("/bookings/create-bookings", () => {
```

Comment:
nit: Like discussed before, we want to test cases we don't want to happen, such as a user being added to the booking twice - it would be nice to have a test case for this